plugins/oauth/web_client/templates/oauth_config.jade

```diff
@@ -1,34 +1,36 @@
 .g-config-breadcrumb-container
 
 .g-oauth-provider-list-title Configure supported providers below
+p Only fill in the information for the OAuth providers you wish to enable.
```

Comment:
We can call the plugin ``oauth``, but we should refer to the protocol as ``OAuth2`` in all user-facing configuration strings.